pontoon/base/models.py

```diff
@@ -451,6 +527,12 @@ class ProjectLocale(models.Model):
     project = models.ForeignKey(Project, related_name='project_locale')
     locale = models.ForeignKey(Locale, related_name='project_locale')
 
+    # Aggregated stats for this project in this locale.
+    total_strings = models.PositiveIntegerField(default=0)
```

Comment:
These fields and the operations around them smell like an abstract model base class you can reuse for all the models in question.